src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -76,6 +77,13 @@ public synchronized void addPerson(Person person) throws UniquePersonList.Duplic
         updateFilteredListToShowAll();
         indicateAddressBookChanged();
     }
+    
+    @Override
+    public synchronized void addTask(Task task) {
+        addressBook.addTask(task);
+        updateFilteredListToShowAll();
+        indicateAddressBookChanged();
+    }
 
```

Comment:
Try to also implement delete/find as requested on V0.1.
V0.2 onwards, deadlines will be strict, thus please attempt to meet them!